subsys/net/ip/connection.c

```diff
@@ -760,8 +760,7 @@ enum net_verdict net_conn_input(struct net_pkt *pkt,
 		} else if (IS_ENABLED(CONFIG_NET_SOCKETS_CAN) && conn_family == AF_CAN) {
 			best_match = conn;
 		}
-			/* loop end */
-	}
+	} /* loop end */
```

Comment:
Really? For a one-liner? Seems like a lot of maintenance overhead for the maintainers (and myself). As an external contributor I'd stop fixing such things if they always had to be done in separate PRs. (While a separate commit is totally fine of course.) If you insist I'll do so but it would be great if you could extend a little on your rationale behind it so it becomes easier to accept this.